client-ts/signalr-protocol-msgpack/spec/MessagePackHubProtocol.spec.ts

```diff
@@ -18,6 +18,10 @@ describe("MessageHubProtocol", () => {
         expect(parsedMessages).toEqual([invocation]);
     });
 
+    it("can read Invocation message with Timestamp argument", () => {
+        const bytes = "lQGAwKZtZXRob2SR1v9Y7ByA";
```

Comment:
Incomplete?

Reply:
Yeah, I changed my plan and just forgot to remove this test.